collection-scripts/gather_network_logs

```diff
@@ -23,6 +23,24 @@ for NODE in $NODES; do
     PIDS+=($!)
     oc -n openshift-sdn exec $OVS_POD -- bash -c "cat /var/log/openvswitch/ovsdb-server.log" > ${NETWORK_LOG_PATH}/${OVS_POD}_ovsdb_log &
     PIDS+=($!)
+
+    oc -n openshift-sdn exec $SDN_POD -- bash -c "ip a" > ${NETWORK_LOG_PATH}/${NODE}_ip &
+    PIDS+=($!)
+
```

Comment:
Do we need to collect this? Isn't this provided by the node information?

Reply:
Node information? Do you mean:
 - oc get node ..
 - oc describe node ..
None of the above will give us the full list of node interfaces.

Reply:
@dcbw or @danwinship (can't remember who) you didn't think this was needed in the initial draft of this script, do you feel this is ok to add?

Reply:
It's not really interesting for IPI OCP 4.1. It might be useful for UPI.